frontend/src/LoadingScreen.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import load from './img/Pulse-1s-200px.gif'
+
+function LoadingScreen() {
+  return (
+    <div id="row">
```

Comment:
id="row"?

Reply:
Arra a div-re nincs is szüksgég, töröltem. Id-t meg elfelejtem most már.